examples/stable/test/end_to_end/candid_rpc/llm/test/tests.ts

```diff
@@ -0,0 +1,17 @@
+import { ActorSubclass } from '@dfinity/agent';
+import { expect, it, Test } from 'azle/test';
+
+import { _SERVICE } from './dfx_generated/agent/agent.did';
+
+export function getTests(actor: ActorSubclass<_SERVICE>): Test {
+    return () => {
+        if (process.env.GITHUB_ACTIONS !== 'true') {
```

Comment:
could we do something like `const test = process.env.GITHUB_ACTIONS === true ? it.skip : it`